docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -155,17 +155,32 @@ can be used only through tsh in [a local terminal tab](#opening-a-local-terminal
 Alternatively, you can look for the application in the search bar and press `Enter` to launch it in
 the browser.
 
+### Connecting to TCP apps with VNet on macOS
+
+VNet is a virtual network emulation tool available in Teleport Connect. It enables any program on
+your computer to connect to TCP applications protected by Teleport, with no extra changes required
+to the program itself. It removes the need to manually create tunnels for TCP applications.
+
+1. Log in to the cluster.
+1. To start VNet, open the connection list in the top left and click the connect icon to the right of
+   VNet.
+1. Now access any of your web and TCP apps over its regular public address.
```

Comment:
Does "any of your web and TCP apps over its regular public address" mean the same thing as "any web or TCP app you can access with Teleport using its public address (a subdomain of your Teleport cluster domain)"?

Reply:
Oops, it shouldn't say "web and TCP", it should be TCP apps only.

For most users it's going to mean the same thing. However, it's possible to set [a custom public address](https://goteleport.com/docs/application-access/guides/connecting-apps/#customize-public-address) for an app that's different than the address of the cluster.

For VNet to work with such apps, some extra setup needs to be done by an admin. This will be covered in another PR.